cmd/init_helm_promise.go

```diff
@@ -108,3 +117,52 @@ func generateResourceConfigurePipeline() (string, error) {
 	}
 	return string(pipelineBytes), nil
 }
+
+func schemaFromChart() (string, error) {
+	values, err := valuesFromChart()
+	if err != nil {
+		return "", err
+	}
+	schema, err := internal.HelmValuesToSchema(values)
+	if err != nil {
+		return "", err
+	}
+
+	bytes, err := yaml.Marshal(*schema)
+	if err != nil {
+		return "", err
+	}
+
+	return string(bytes), nil
+}
+
+func valuesFromChart() (map[string]interface{}, error) {
+	client, err := helmclient.New(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	install := action.NewInstall(&action.Configuration{})
+	registryClient, _ := registry.NewClient()
+	install.SetRegistryClient(registryClient)
+
+	if chartName != "" {
+		install.RepoURL = chartURL
+	}
+
+	chart, _, err := client.GetChart(getChartName(), &install.ChartPathOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	return chart.Values, nil
+}
+
```

Comment:
shall we do some more error wrapping? I think a lot of these would be confusing to the user/us without some additional context

Reply:
done